### ISSUE TITLE
Feature Request: Support calling ethr as a library

### ISSUE THREAD

**Issue by Michaelhobo** [microsoft/ethr#118]
Would you be open to extracting some sort of API for non-CLI use cases? Perhaps extract the business logic into a library and have the CLI options just change the parameters going into the library.

**Comment by pgmsft** [comment]
Yes, that would be really good. If you can submit a patch, I can incorporate it.
